### PR TITLE
Add new hexagons for projects lyra and bolt

### PR DESCRIPTION
**Pull request by ahpook**
These are puppet-related projects that we're printing
physical hex stickers for!

### CODE REVIEW

**Comment by hexagontruth** [comment]
Hi! Several things:

1. I think your SVGs are somehow malformed — neither of them are displaying correctly for me.
2. Why is the Bolt logo shrunk down? This will technically work but I think the preference is for it to fill the entire hex, with a background (presumably white in this case) if it is indeed mean to display at this size.
3. Not a big deal since it will be rescaled via CSS, but the Lyra png is 180x209, and Bolt is 181x208. The correct size is 181x209. (I didn't pick this ratio, but assume it was chosen because 209 * sqrt(3) / 2 — the apothem of a hexagon — is relatively close to a whole number.)
4. Again not a big deal, but if you are going to sell or distribute these have you thought of including the `order_online_url` in the metadata?

Anyway tl;dr please fix at least the SVGs. Thanks!

**Review by hexagontruth — CHANGES_REQUESTED** [review]
Please fix the SVGs.

Consider also scaling the Bolt logo to fill its respective hex, and rescaling both images to 181x209.

**Comment by ahpook** [comment]
Thanks for the feedback @hexagrahamaton - i pushed an amended commit with correctly-sized PNGs and fixed the SVGs. I'd like to keep the bolt logo the current size with the border because it helps with the "3d" effect. Thank you!!

**Comment by hexagontruth** [comment]
Can you elaborate on what you mean by the 3D effect?

**Comment by ahpook** [comment]
@hexagrahamaton just that the border between the edge of the hex and the bolt logo is the same width as the lines of the logo itself - so it pulls your eye to the "missing" corner of the 3d box, highlighted in yellow.  🤷‍♂ no biggie
bolt-dot

**Comment by hexagontruth** [comment]
Done, but I'm not 100% sure we're talking about the same thing. I've attached a screenshot — please review at your convenience. (At some point I'd like, if the other maintainers agree, an opaque background for the hexes — at which point the empty hexagonal ring around your sticker will appear, idk, probably white or something.)
boltsticker

**Comment by ahpook** [comment]
Ah I see what you mean. I'll work on this with the designer who made the tiles. Thanks for the merge!
